### PR TITLE
Wirvsvirus

### PR DESCRIPTION
**Pull request by stephanflemming**
add news and event

### CODE REVIEW

File: _events/2020-03-19-wirvsvirus.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: "WirVsVirus Hackathon"
+starts: 2020-03-20
+ends: 2020-03-22
+organiser:
+  name: Bundesregierung
+  email: info@wirvsvirushackathon.org
+location: online
+---
+
+The Federal Government of Germany is organising the [#WirVsVirus Hackathon](https://wirvsvirushackathon.org/).
```
Comment (bgruening) [inline _events/2020-03-19-wirvsvirus.md]: a little bit more encouraging ... please feel free to join ... etc... We offer jupyter notebook, RStudio instances, link to the training mterial about it ...

Link to this project and more importantly to the data ...
https://covid19.galaxyproject.org/

Comment (bgruening) [reply _events/2020-03-19-wirvsvirus.md]: please add add the support email adress if people need resources they should contact us ...

Comment (gmauro) [reply _events/2020-03-19-wirvsvirus.md]: We have only 32 slots free for IT. Should I start to free other cloud nodes?

File: _posts/2020-03-19-wirvsvirus.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+site: freiburg
+title: "WirVsVirus Hackathon"
+tags: [hackathon]
+---
+
+The Federal Government of Germany is organising the [#WirVsVirus Hackathon](https://wirvsvirushackathon.org/).
+
+![WirVsVirus](https://wirvsvirushackathon.org/wp-content/uploads/2020/03/12-scaled.jpg)
+
+We want to encourage you to participate and join the fight against Covid-19.
```
Comment (mtekman) [inline _posts/2020-03-19-wirvsvirus.md]: I think this should be the first paragraph

File: _posts/2020-03-19-wirvsvirus.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+site: freiburg
+title: "WirVsVirus Hackathon"
+tags: [hackathon]
+---
+
+The Federal Government of Germany is organising the [#WirVsVirus Hackathon](https://wirvsvirushackathon.org/).
```
Comment (mtekman) [inline _posts/2020-03-19-wirvsvirus.md]: I think this should be the second paragraph

"The Federal Government....., and the GalaxyProject is here to help."

**Comment by mtekman** [comment]
```
---
site: freiburg
title: "WirVsVirus Hackathon"
tags: [hackathon]
---

We want to encourage you to participate and join the fight against Covid-19.

The Federal Government of Germany is organising the [#WirVsVirus Hackathon](https://wirvsvirushackathon.org/), and we at Galaxy are here to help.

![WirVsVirus](https://wirvsvirushackathon.org/wp-content/uploads/2020/03/12-scaled.jpg)


The Galaxy community freely offers [RStudio](https://galaxyproject.github.io/training-material/topics/galaxy-ui/tutorials/rstudio/tutorial.html) and [Jupyter notebook](https://galaxyproject.github.io/training-material/topics/galaxy-ui/tutorials/galaxy-intro-jupyter/tutorial.html) instances on request, hosted using our cloud infrastructure aided by more than 3000 cores and extensive memory resources.

All tools and notebook instances are backed by a foundation of scientific and computational reproducibility, using the power of [Bioconda](https://bioconda.github.io/), along with the full [Galaxy suite](https://usegalaxy.eu) of 2000 bioinformatic tools to experiment with at your disposal.

Take a look at https://covid19.galaxyproject.org/ for an initial analysis of COVID-19 data using Galaxy, BioConda and public research infrastructure (XSEDE, de.NBI-cloud, ARDC cloud).

Feel free to [contact us](galaxy@informatik.uni-freiburg.de) for further information or if we can support your research with our infrastructure.

Stay healthy!
```

I've never understood how to push edits to a checked out PR from the command line.

**Comment by stephanflemming** [comment]
I use the same text for the news and event section, or?

**Comment by mtekman** [comment]
sounds good to me

**Comment by bgruening** [comment]
Thanks all.

**Pull request merged**